### PR TITLE
added measurements to availability and dependency, renamed dependency…

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
1. added `measurements` to availability and dependency
2. updated availability schema
3. renamed `dependencyTypeName` to `type`
4. deprecated `handledAt`

### CODE REVIEW

File: Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs [hunk 1]
```diff
@@ -38,9 +38,9 @@ public void SerializeWritesNullValuesAsExpectedByEndpoint()
             DependencyTelemetry original = new DependencyTelemetry();
             original.Name = null;
             original.Data = null;
-            original.DependencyTypeName = null;
+            original.Type = null;
             original.Success = null;
-            original.DependencyTypeName = null;
+            original.Type = null;
```
Comment (dnduffy) [inline Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs]: Why is there a duplicate of this line two lines up?

**Comment by dnduffy** [comment]
:shipit:

File: Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs (hunk 1 above)
Comment (SergeyKanzhelev) [inline Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs]: thanks. fixed. There is a duplication as there used to be setting of both - `DependencyKind` and `DependencyType`. Now it should only be `Type`

**Comment by SergeyKanzhelev** [comment]
Tests failing as Breeze do not support some of those changes yet. Working on it

**Pull request merged**